.travis.yml

```diff
@@ -1,12 +1,23 @@
 sudo: required
 
+before_install:
+    - sudo apt-add-repository -y ppa:ubuntu-toolchain-r/test
+    - sudo apt-add-repository -y ppa:beineri/opt-qt542
+    - sudo apt-get -qq update
+    - sudo apt-get -qq install g++-4.8 libc6-i386 qt54tools qt54svg qt54webkit qt54serialport qt54multimedia
+    - PATH="/opt/qt54/bin:$PATH"
+    - qt54-env.sh
+
 install:
-    - wget 'https://sourceforge.net/projects/astyle/files/latest/download'
+    - wget 'https://sourceforge.net/projects/astyle/files/astyle/astyle%202.06/astyle_2.06_linux.tar.gz/download'
     - tar -zxvf download
     - (cd astyle/build/gcc && make release && sudo make install)
     - rm astyle -rf
 
 script:
+    - qmake
+    - make
+    - cd ..
```

Comment:
Right now, Travis is unable to find `astylerc`. I think that's because you moved out of the directory, so delete this and it should be good.